packages/@okta/vuepress-site/docs/guides/manage-orgs-okta-aerial/main/index.md

```diff
@@ -0,0 +1,377 @@
+
+```bash
+Authorization: Bearer ${access_token}
+```
+<!-- our OAuth docs for service apps rely on Postman for this step. need something in the interim until Postman is ready -->
+
+## Create a child Org
+
+Create a child org using the Org creator API credentials in the parent org. This creates a child org with features synced from the parent org. In the API response, you receive an API token tied to the super admin. Use the token to provision more resources on the child org, like policies, apps, or groups.
+
+This isn't the token that is used for Okta Aerial. See the [Org creator API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/OrgCreation/). The API token that the Org creator API creates has the same automatic expiration and deactivation as API tokens created using the [Admin Console](/docs/guides/create-an-api-token/main/#token-expiration-and-deactivation).
+
+However, the Org creator API token doesn’t appear in the Admin Console. You can’t use the console to revoke the token. If you deactivate the super admin (the first admin created during org creation), the token is deactivated.
+
+If you lose this token or it auto-expires, you must log in to the Admin Console as a Super Admin and [create a token](/docs/guides/create-an-api-token/main/#create-the-token).
+
+
+## Link the Org to the Aerial account
+
+To enable products in an org, add the org to your Aerial account. You can only add orgs to your Aerial account that are associated with your Okta contracts.
+
+To link the org to Okta Aerial, send a `POST` request to the `/api/va/orgs` endpoint of the Aerial API. The response contains the Org object including the `orgId`. Use the `orgId` to enable products.
+
+### Use `orgId`
+
+<ApiOperation method="post" url="https://aerial-{region}/{accountId}/api/v1/orgs" />
+
+```bash
+Authorization: Bearer ${access_token}
+
+{
+  "orgId": "00oy0itaI2Yi7XGGE0g3",
+  "cell": "OK1"
+}
+```
+
+### Use `domain`
+
+<ApiOperation method="post" url="https://aerial-{region}/{accountId}/api/v1/orgs" />
+
+```bash
+Authorization: Bearer ${access_token}
+
+{
+  "domain": "${yourOktaDomain}",
+  "cell": "OK1"
+}
+```
+
+### Response example
+
+<!-- We have this note in the Google doc. What to do?
+
+Note: this API is still in development. We will update the documentation with more accurate examples as we progress in development. -->
+
+The id of this record is the `orgId` to use in the URL for enabling Products:
+
+```json
+{
+  "accountId": "string",
+  "name": "string",
+  "region": "string",
+  "cell": "string",
+  "domain": "string",
+  "status": "string",
+  "createdDate": "string",
+  "createdBy": "string",
+  "id": "string"
+}
+```
+
+## Enable products in the Org
+
+<!-- should add a bit of concept content here -->
+
+<ApiOperation method="put" url="https://aerial-{region}/{accountId}/api/v1/orgs/{orgId}/products" />
+
+### Request and response example
+
```

Comment:
What is this a request and response example of? It looks like the same as the response example just above and doesn't look like a request example...

Reply:
I've fixed this. The request example should be an array of products. If a product is in the array, it's enabled on the org. If it's not included in the array, it's disabled from the org.

To make sure it's clear, I've separated them into a request and a response example, even though they're basically the same details.